packages/apps-config/src/ui/colors.ts

```diff
@@ -247,10 +248,13 @@ export const chainColors: Record<string, string> = [
   ['Westlake', chainWestlake],
   ['Zenlink PC1', chainZenlink],
   ['ZERO.IO', chainZero]
-].reduce((colors, [chain, color]): Record<string, string> => ({
-  ...colors,
-  [chain.toLowerCase()]: color
-}), {});
+].reduce(
+  (colors, [chain, color]): Record<string, string> => ({
+    ...colors,
+    [chain.toLowerCase()]: color
+  }),
+  {}
+);
```

Comment:
Not sure why these changes slipped in in all 3 places.

Reply:
running `yarn lint` seems to change it. It could be an issue with my local config.

Reply:
Changed the indents